### PR TITLE
Eclipse Foundation Adaptation: fix tests failing

### PR DESCRIPTION
**Pull request by fatadel**
Continuation of #457

### CODE REVIEW

File: .github/workflows/ci-cd.yaml [hunk 1]
```diff
@@ -1,66 +1,64 @@
...
-      
-      - name: TM test
-        run: node packages/cli/index.js -t TM -i examples/tm/*/*.jsonld
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v1
+              with:
+                  node-version: ${{ matrix.node-version }}
 
-  deploy-via-ssh:
+            - name: TD test
```
Comment (egekorkan) [inline .github/workflows/ci-cd.yaml:33]: this shouldn't be before installing lerna

**Review by egekorkan — CHANGES_REQUESTED** [review]
also we need the license notice on top of the code files

**Comment by fatadel** [comment]
> also we need the license notice on top of the code files

Oh, indeed!

**Comment by fatadel** [comment]
The license headers are added using [this VSCode plugin](https://marketplace.visualstudio.com/items?itemName=angelo-breuer.license-header-manager).

**Review by egekorkan — APPROVED** [review]

**Pull request merged**